spec/features/admin/managing_user_spec.rb

```diff
@@ -15,23 +15,23 @@
       end
 
       it "allows admin to edit internal/external pricing" do
-        expect(page).to have_content("Internal Pricing")
+        expect(page).to have_content("Internal Pricing\n")
```

Comment:
I get the modifications below, but it seems strange that `Internal Pricing` would fail, and `Internal Pricing\n` would pass. Oh, poltergeist…

Reply:
Good catch. I think this one was probably a find/replace all accident that isn't necessary. I'll try changing that.